### PR TITLE
setting up repository

### PR DESCRIPTION
**Pull request by omcmenamin**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
```
Comment (glenn-hamilton) [inline app.ts]: The `package.json` and `package-lock.json` files are missing

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        "target": "ES5",
+        "sourceMap": true,
+        "noImplicitAny": true
+    },
+    "include": ["./**/*"],
+    "exclude": [
+        "./node_modules/**/*",
+        "./dist/**/*"
+    ]
+}
```
Comment (glenn-hamilton) [inline tsconfig.json]: Very minor issue, but each of the files should have a new empty line at the end

File: app.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+const session =require('express-session')
+//const { title } = require('process');
```
Comment (glenn-hamilton) [inline app.ts]: Don't need this

File: app.ts (hunk 1 above)
Comment (omcmenamin) [reply app.ts]: Mickey has updated these issues

File: app.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+const session =require('express-session')
```
Comment (glenn-hamilton) [inline app.ts]: This import is unused

File: app.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+//const { title } = require('process');
+
+const app = express();
+
+//configure nunjucks
+const appViews = path.join(__dirname, '/views');
```
Comment (glenn-hamilton) [inline app.ts:9]: Can you add a basic HTML page to show that the project will work correctly

**Review by glenn-hamilton — CHANGES_REQUESTED** [review]

File: app.ts (hunk 4 above)
Comment (mickey-dawson) [reply app.ts]: > This import is unused

which import specifially is unused?

Comment (glenn-hamilton) [reply app.ts]: `session`

Comment (mickey-dawson) [reply app.ts]: login html is being worked on, we just need these changes approved first

File: app.ts (hunk 5 above)
Comment (mickey-dawson) [reply app.ts:9]: > Can you add a basic HTML page to show that the project will work correctly

HTML Login page is being worked on, we just need these changes approved first. I also hotfixed the issues with app.ts if you want to review those

File: HelloWordl.html [hunk 6]
```diff
@@ -0,0 +1 @@
+<p>Hello Wordle!</p>
```
Comment (glenn-hamilton) [inline HelloWordl.html]: This needs to go in a `views` folder

File: app.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+//const { title } = require('process');
```
Comment (glenn-hamilton) [inline app.ts]: `//const { title } = require('process');` is unused

File: app.ts [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+app.set('view engine', 'html');
+
+app.use('/public', express.static(path.join(__dirname, 'public')));
+
+app.use(express.json())
+
+app.use(express.urlencoded({ extended: true }));
+
+app.listen(3000, ()=> {
+    console.log('Server listening on port 3000');
```
Comment (glenn-hamilton) [inline app.ts:30]: This still needs a basic route to test that the page works

Comment (mickey-dawson) [reply app.ts:30]: > This still needs a basic route to test that the page works

Set it up there now and pushed it

**Review by MylesScullion — APPROVED** [review]
Look correct to me

**Review by MylesScullion — APPROVED** [review]
Looks good

**Review by MylesScullion — APPROVED** [review]

**Review by mickey-dawson — APPROVED** [review]

**Review by glenn-hamilton — APPROVED** [review]

**Pull request merged**